scene/2d/canvas_item.cpp

```diff
@@ -1088,8 +1091,9 @@ void CanvasItem::_bind_methods() {
 	ADD_GROUP("Material", "");
 	ADD_PROPERTYNZ(PropertyInfo(Variant::OBJECT, "material", PROPERTY_HINT_RESOURCE_TYPE, "ShaderMaterial,CanvasItemMaterial"), "set_material", "get_material");
 	ADD_PROPERTYNZ(PropertyInfo(Variant::BOOL, "use_parent_material"), "set_use_parent_material", "get_use_parent_material");
+	ADD_GROUP("Context", "");
+	ADD_PROPERTYNZ(PropertyInfo(Variant::BOOL, "inherits_transform", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_EDITOR), "set_inherits_transform", "get_inherits_transform");
```

Comment:
Remove the `, PROPERTY_HINT_NONE, "", PROPERTY_USAGE_EDITOR` part, since the defaults are better than this. Currently, this won't save the flag in the scene.

Also, it should be `ADD_PROPERTYNO`, so it saves a false value, but not a true value. (NZ is not-zero, NO is not-one)

Reply:
Ah, that makes sense. I made the change in both places.